client/components/Layout/styles.ts

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+export const StyledLayout = styled.div`
```

Comment:
why we don't need it? if it just for layouting, I think making a new style while we have grid by materialUi which CSS framework that we use, is unnecessary. IMHO

could you elaborate more why we don't need it?

Reply:
It's only an MVP right? We can add this in later at anytime pretty easily if we decide that it's necessary but I don't think that we should pull in the Material UI grid just for the sake of using everything from the UI library. I wouldn't delay merging the layout PR if it already gets the job done.

Reply:
We need to keep a flexible grid that ensures consistency across layouts, containing details about how content reflows on different screens and providing ease of how an app can scale from small to extra-large screens. At this moment, after reviewing the UI that we have designed thus far, the simplicity that they have, which is the vision that we have, to keep simple, it is better to use the Material UI since it constraints styling to their existing guidelines.  For example: 
Inline-style: 
![alt text](https://media.discordapp.net/attachments/633826607617933353/644814067239878656/Screenshot_43.png "Simple")
![image](https://user-images.githubusercontent.com/36430592/68989759-06d44280-0819-11ea-9108-662812ed493b.png)
Here is some helpful documentation: [Grid](https://material-ui.com/components/grid/)